pages/studentPortal/web/React/components.tsx

```diff
@@ -1,21 +1,26 @@
 import { useRouter } from "next/router";
-import React from "react";
 import LessonComponent, {
   LessonComponentData,
 } from "../../../../components/studentPortal/lessons/LessonComponent";
 import { Button } from "../../../../components/ui/Button";
+import { getComponentsLessonData } from "../../../api/studentPortal/web/react/components";
+
+const React2 = (props: { lessonComponents: LessonComponentData[] }) => {
```

Comment:
so if the api is working correctly, we should be able to delete this file in favour of using this dynamic component:

https://github.com/skillify-ca/skillify-web/blob/main/pages/studentPortal/intro/%5B...slug%5D.tsx#L65

Currently that dynamic lesson component is hard-coded to the basics course. We will need to use nextjs dynamic routing to support a dynamic course and lesson name.

https://nextjs.org/docs/routing/dynamic-routes

This file will have to be moved from `pages/studentPortal/intro/[...slug]` and to `pages/studentPortal/[courseId]/[lessonId]`